tests/admin/lmod/LICENSE

```diff
@@ -0,0 +1,24 @@
+Copyright (c) 2015, Intel Corporation
```

Comment:
This license file seems unnecessary.

Reply:
Removed!
Copy/paste from the spack/ folder